src/keychain/index.ts

```diff
@@ -374,13 +384,15 @@ export async function setSharedWebCredentials(username: string, password: string
 export async function getPrivateAccessControlOptions(): Promise<SetOptions> {
   logger.debug(`[keychain]: getPrivateAccessControlOptions`, {}, logger.DebugContext.keychain);
 
-  const isSimulator = IS_DEV && (await DeviceInfo.isEmulator());
+  const isSimulator = IS_DEV && IS_IOS && (await DeviceInfo.isEmulator());
 
   if (isSimulator) return {};
 
+  const usePin = await shouldAuthenticateWithPIN();
+
   return {
     accessControl: ios ? ACCESS_CONTROL.USER_PRESENCE : ACCESS_CONTROL.BIOMETRY_CURRENT_SET_OR_DEVICE_PASSCODE,
     accessible: ACCESSIBLE.WHEN_UNLOCKED_THIS_DEVICE_ONLY,
-    storage: STORAGE_TYPE.RSA,
+    storage: usePin ? STORAGE_TYPE.AES_GCM_NO_AUTH : STORAGE_TYPE.RSA,
```

Comment:
Could you add a comment with:
`storage` is now explicitly set in order to specify RSA instead of the symmetric AES_GCM default.